src/lib/backing-types/find.ts

```diff
@@ -0,0 +1,5 @@
+import { findFiles } from '../../utils/path'
+
+export function find(pattern?: string): Promise<string[]> {
+  return findFiles(pattern ?? './**/*.ts')
```

Comment:
This should only find files within layout.sourceDir` right?

Reply:
Maybe though enforcing that is at a higher level. Would make sense. Avoiding cross-lib deps where possible.